Cargo.toml

```diff
@@ -19,6 +19,7 @@ circle-ci = { repository = "tendermint/kms" }
 abscissa = "0.0.6"
 abscissa_derive = "0.0.2"
 byteorder = "1.2"
+base64 = "0.10.0"
```

Comment:
Oh thanks. Should have asked directly. I spent half an hour or so searching for a light weight base64 crate.

Reply:
That crate is probably fine now, but [had some issues in the past](https://github.com/alicemaz/rust-base64/issues/29). Regardless, it'd probably be good to avoid adding additional dependencies.

Replacing it should be as easy as:

```rust
use subtle_encoding::base64
```

...and it should otherwise be (mostly) API-compatible.

Reply:
done

Reply:
Awesome, looks good!

Also: this is literally the use case `subtle-encoding` is designed for: decoding private keys in constant time.